geode-experimental-driver/src/main/java/org/apache/geode/experimental/driver/DriverFactory.java

```diff
@@ -89,4 +89,36 @@ public DriverFactory setPassword(String password) {
     this.password = password;
     return this;
   }
+
+  /**
+   * Specifies the key store to use with SSL.
+   *
+   * @param keyStorePath Path to the SSL key store.
+   * @return This driver factory.
+   */
+  public DriverFactory setKeyStorePath(String keyStorePath) {
```

Comment:
Really seems like there should just be a single function setSSLPaths(keyStorePath, trustStorePath), as it doesn't seem to be valid to just call one of these functions.  That being said, that might not plug as nicely into the builder pattern that's used here, so...your call.